python_modules/dagster-graphql/dagster_graphql/schema/asset_checks.py

```diff
@@ -118,6 +110,9 @@ def __init__(
         self.timestamp = execution.create_timestamp
         self.stepKey = execution.event.step_key if execution.event else None
 
+    def resolve_status(self, graphene_info: "ResolveInfo") -> AssetCheckExecutionResolvedStatus:
+        return self._execution.resolve_status(graphene_info.context)
```

Comment:
lazily resolve the status as this requires making a database call (and it's possible the status won't be needed!).

mechanically, we also just have to wait because we need the LoadingContext to be available in order for us to cache calls.